src/graph.cpp

```diff
@@ -261,7 +260,7 @@ std::shared_ptr<Edge> Graph::findEdge(const unsigned int &v1id, const unsigned i
 
 /**
  * A backtracking function for getting the value of the smallest weighted path for the given graph
```

Comment:
I don't think this is very well explained, you can refer directly to the tsp problem, and also something like a closed route or even better hamiltonian cicle to indicate it visits everything once and only once, and loops back to the beggining